cypress/support/commands.js

```
@@ -1,25 +1,19 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-//
-//
-// -- This is a parent command --
-// Cypress.Commands.add('login', (email, password) => { ... })
-//
-//
-// -- This is a child command --
-// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
-//
-//
-// -- This is a dual command --
-// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
-//
-//
-// -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+// Command to log in
+Cypress.Commands.add(
+  'login',
+  (username = 'tomsmith', password = 'SuperSecretPassword!') => {
+    cy.visit('/login');
+    cy.get('#username').clear();
+    cy.get('#username').type(username);
+
+    cy.get('#password').clear();
+    cy.get('#password').type(password);
+
+    cy.get('button[type="submit"]').click();
+  }
+);
+
+// Command to log out
+Cypress.Commands.add('logout', () => {
+  cy.get('a[href="/logout"]').click();
```

Good: The logout command clicks the logout link `a[href="/logout"]`, which matches the UI on the secure page. Consider (optionally) adding a safety assertion before clicking (e.g., `cy.get('a[href="/logout"]').should('be.visible')`) to fail early with a clearer message if the app isn't in an authenticated state, but this is not required for functionality.